modules/gitops-argo-cd-properties.adoc

```diff
@@ -11,19 +11,24 @@ The Argo CD Custom Resource consists of the following properties:
 [options="header"]
 |===
 |Name |Description |Default |Properties
+
 |`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
 
-|`applicationSet`
-|`applicationSet` controller configuration options.
-| `_<object>_`
+|`aggregatedClusterRoles`    |The aggregation of several `ClusterRoles` for a cluster-scoped instance. | `__<empty>__` |
```

Comment:
Fields should be in alphabetical order, AFAICT.

Reply:
Fixed.